geode-core/src/main/java/org/apache/geode/management/CacheServerMXBean.java

```diff
@@ -60,48 +61,48 @@
   /**
    * Returns the port on which this CacheServer listens for clients.
    */
-  public int getPort();
+  int getPort();
```

Comment:
Why are we removing public from all of these methods?  I think CacheServerMXBean is a public facing API and intends to expose these methods.